java/piranha/src/main/java/com/uber/piranha/PiranhaUtils.java

```diff
@@ -44,4 +48,78 @@ public static boolean isPrefixPath(TreePath prefix, TreePath path) {
     }
     return false;
   }
+
+  /**
+   * Returns a matcher for the argument passed to a new class creation.
+   *
+   * @param index of the argument to be matched
+   * @param argMatcher The matcher for the argument
+   * @return Returns true of the argument at index matches argMatcher, otherwise false.
+   */
+  public static Matcher<NewClassTree> newClassHasArgument(
+      int index, Matcher<ExpressionTree> argMatcher) {
+    return new NewClassArgument(index, argMatcher);
+  }
+
+  /**
+   * Returns a matcher for the arguments passed to a new class creation.
+   *
+   * @param matchType - all, any, none
+   * @param argMatcher The matcher for the argument
+   * @return Returns true if any (or all or none) arguments match, otherwise false.
```

Comment:
Super small nit: "if all (or any or none)", just to keep the same order as in the documentation of `matchType`

Reply:
Again, this one doesn't matter a whole lot, but I don't think it was "resolved" per se. My suggestion was to list the match types in the same order for `@param matchType` and for `@return`.